backend/src/test/resources/application-default.yaml

```diff
@@ -1,11 +1,24 @@
 spring:
   profiles.include: no-security,no-okta-mgmt, no-okta-auth, no-experian
   datasource:
-    url: jdbc:postgresql://localhost:${test-db-port:5444}/simple_report
-    hikari.minimum-idle: 2 # when a context isn't being used, it should not leave lots of connections lying around
+    simplereport:
+      hikari:
+        jdbc-url: jdbc:postgresql://localhost:${test-db-port:5444}/simple_report
+        minimum-idle: 2
+        idle-timeout: 10000
```

Comment:
By setting this here, will this value propagate to the production instance, since it is not overridden there? Setting this value so low can cause a significant overhead as new connections are established, so I want to make sure it is intentional, and won't go further than we want it to.

Reply:
This is actually the test version so we should be good. The dual data sources cause tests to hang on to more connections than they were previously and tests were failing due to max connections being reached on the Docker Postgres instance (100), so the test properties manages them fairly aggressively. None of the existing non-test settings should be changed, they're just indented a level.